.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Question via Jaeger Community Support (online chat)
+    url: https://cloud-native.slack.com/archives/CGG7NFUJ3
```

Comment:
Some people get confused by you can't create an account at this link. We either need to include the other link that creates invites or point to a page that has both (like on the website)